steam_wrapper/steam_wrapper.py

```diff
@@ -146,9 +149,12 @@ def migrate_data():
     """
     source = os.path.expandvars("$XDG_DATA_HOME")
     target = DATA
+    backup = f'{DATA}.bak'
     steam_home = os.path.join(source, "Steam")
     xdg_data_home = os.path.join(STEAM_ROOT, target)
     if not os.path.islink(source):
+        if os.path.isdir(target):
+            copytree(target, backup)
```

Comment:
It's not possible for the Steam directory to be in `~/.local/share/Steam` before migration, right? Otherwise, it should be `ignore=`'d here.

Reply:
Should not be. These are paths inside sandbox

Reply:
I can imagine 2 scenarios how it can happen:
- Steam client ignored `$XDG_DATA_HOME` and created its data dir under the fallback path `~/.local/share` (older Steam client didn't respect XDG spec?)
- `~/.var/app/com.valvesoftware.Steam/data` symlink got deleted or not created (e.g. after interrupted migration)